Sources/StreamChat/ChatClient.swift

```diff
@@ -206,6 +210,14 @@ public class ChatClient {
             databaseContainer,
             apiClient
         )
+        memberUpdater = environment.memberUpdaterBuilder(
```

Comment:
Feels a bit strange to have member updaters in the chat client. Although they don't depend on anything `Chat` specific, I believe we can instantiate them on demand - no need to move them here. Wdyt?

Reply:
I'll get rid of these. These should be converted to a repository class anyway at some point.